### DISCUSSION TITLE
전역상태관리 관련해서 어떤 패키지를 사용할지 고민입니다.

### DISCUSSION THREAD

**Discussion by HJ97Kim** [MoCo-v2/moco-client#4] [Q&A] [answered]
전역상태관리 관련해서 어떤 패키지를 사용할지 고민입니다.
되도록이면 가벼운 패키지로 사용하고 싶어요.

**Comment by wlswo** [comment, marked as answer]
zustand 사용해보심이 어떤가 싶습니다. 가볍기도 하고 최근 네이버 메인사이트도 zustand 로 바꾸었더라구요 😀
